scripts/scil_filter_tractogram_anatomically.py

```diff
@@ -118,17 +207,27 @@ def main():
     o_dict['streamline_count_after_filtering_length'] =\
         len(length_sft.streamlines)
 
+    if args.save_outliers:
+        outlier = compute_outliers(sft, length_sft)
+        o_dict['streamline_count_discarded_when_filtering_length'] =\
+            len(outlier.streamlines)
+        save_outliers(out_path, 'length', 'length', in_tracto_name, ext,
+            outlier, length_sft)
```

Comment:
PEP 8: continuation line under-indented for visual indent

Reply:
Done in dbd0b2a.